### PR TITLE
Create OSS Workflow Diagram

### LINKED ISSUE #210: Create OSS Workflow Diagram
## Description
As a contributor,
I need create an OSS workflow Diagram,
so that I can help newcomers know the workflow.

## Mocks
screen shot 2018-06-12 at 10 38 04 am
screen shot 2018-06-12 at 10 38 12 am

### Update [Required]

- [ ] newcomers.component.html
- [ ] newcomers.component.ts

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
4 hours (Trying to find code for curved arrows + Trying to find a way to write modular code for curved arrow alignment) + 7 hours (Tried to fix alignment in mobile phase, find alternate way to represent the information, implement it)

@ancyp @Janiceilene @divyanshu-rawat Can you please assign me the issue? :)

### PR DESCRIPTION
**Pull request by BaaniLeen**
### Description
Created OSS workflow diagram to represent the various stages.
Dependency PR #211 

Fixes #210 

### Type of Change:
**Delete irrelevant options.**

- Code
- Quality Assurance
- User Interface

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)
- This change requires a documentation update (software upgrade on readme file)
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### How Has This Been Tested?
It has been tested locally on localhost. It is mobile responsive.

### Checklist:
**Delete irrelevant options.**

- [ ] My PR follows the style guidelines of this project
- [ ] I have performed a self-review of my own code or materials
- [ ] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] Any dependent changes have been merged 
  

**Code/Quality Assurance Only**
- [ ] My changes generate no new warnings 
- [ ] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been published in downstream modules

### CODE REVIEW

File: src/app/newcomers/newcomers.component.html [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+          <img class="card-img-top" src="{{det.image}}" alt="{{det.methodName}}">
+          <div class="card-body">
+            <h5 class="card-title">{{det.methodName}}</h5>
+            <button type="button" class="btn btn-primary waves-light mx-auto" (click)="style.show()" mdbWavesEffect>View Details</button>
+            <!-- Modal -->
+            <div mdbModal #style="mdb-modal" class="modal fade" id="centralModalSuccess" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"
+              aria-hidden="true">
+              <div class="modal-dialog modal-notify modal-success" role="document">
+                <div class="modal-content">
+                  <div class="modal-header" style="background-color: #8d2165;">
```
Comment (ancyp) [inline src/app/newcomers/newcomers.component.html]: move to scss

File: src/app/newcomers/newcomers.component.html [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+      <div class="row text-center justify-content-center">
+        <h2>OSS Workflow Diagram</h2>
+      </div>
+      <div *ngFor="let stage of workflow_stages">
+        <div class="row text-center justify-content-center mt-5">
+          <button type="button" class="btn btn-link mx-auto" (click)="style.show()" mdbWavesEffect>
+            <h3>{{stage.name}}</h3>
+          </button>
+          <!-- Modal -->
+          <div mdbModal #style="mdb-modal" class="modal fade" id="centralModalSuccess" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"
```
Comment (ancyp) [inline src/app/newcomers/newcomers.component.html]: more appropriate name than style?

Comment (BaaniLeen) [reply src/app/newcomers/newcomers.component.html]: Can you explain this more? :)

**Comment by BaaniLeen** [comment]
@ancyp I have incorporated the suggestions. Could not understand what a more appropriate name than style meant. Can you please explain on that? :)

File: src/app/newcomers/newcomers.component.html (hunk 2 above)
Comment (ancyp) [reply src/app/newcomers/newcomers.component.html]: I assume style is the name you are using to refer to the modal on line 73 in which case a more appropriate name than style would be something that is not generic and sounds like a keyword.

**Comment by ancyp** [comment]
@BaaniLeen resolve  conflicts please.

**Comment by ancyp** [comment]
@BaaniLeen Once the button is done for attracting clicks as discussed in the meeting yesterday, we'll merge this.

**Comment by BaaniLeen** [comment]
@ancyp Conflicts resolved. #Style renamed as well as clickable UI for buttons added. :)

**Pull request merged**